spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -106,7 +106,7 @@
         expect(user.reload.mobile).to_not eq '+1 (555) 555-1212'
       end
 
-      context 'user enters a valid number then changes their mind', sms: true do
+      context 'user enters a valid number then changes their mind' do
         before do
           sign_up_with_and_set_password_for('test@example.com')
           @user = User.find_by_email('test@example.com')
```

Comment:
we can replace instances of:
```
sign_up_with_and_set_password_for('test@example.com')
@user = User.find_by_email('test@example.com')
```
with:
`@user = sign_up_with_and_set_password_for`